rviz_common/CMakeLists.txt

```diff
@@ -246,6 +246,7 @@ ament_target_dependencies(rviz_common
   rclcpp
   resource_retriever
   rviz_assimp_vendor
+  rviz_rendering
```

Comment:
Interesting.  I'm not opposed to doing this, but theoretically this shouldn't be needed; we have `rviz_rendering::rviz_rendering` as part of `target_link_libraries` above.  Any ideas on why that is not sufficient?

Reply:
It seems more "ament-like" to do this and remove the target from `target_link_libraries` above. But I also do wonder why the existing logic is not working...

Reply:
ah, so the version of rviz_rendering released into Rolling does not have the API `rviz_rendering::OgreLogging::get()`. So it must be that the rviz_common is not picking up the version in the workspace. FWIW, I also experience this build error.

Reply:
I guess using ament_target_dependencies causes a re-ordering of compile flags that happens to make it work.